tasks/configure.yml

```diff
@@ -120,6 +130,11 @@
   when: (item.path | basename) not in haproxy_listen_file_list
   with_items: "{{ directory_contents.files }}"
 
+- name: 'Validate listen sections'
+  ansible.builtin.fail:
+    msg: "Listen sections description use undefined variable : {{ haproxy_listen }}"
```

Comment:
```suggestion
    msg: "Listen sections description use undefined variable: {{ haproxy_listen }}"
```

Reply:
I think the error message will always be Listen sections description use undefined variable: VARIABLE IS NOT DEFINED

I personaly prefer to point to the variable name (so : "Listen sections description use undefined variable: (haproxy_listen)")

Reply:
The name of the undefined variable is hidden in the content of haproxy_listen. It's not haproxy_listen variable.